internal/binarylog/env_config.go

```diff
@@ -138,8 +138,9 @@ var (
 	headerMessageConfigRegexp = regexp.MustCompile(headerMessageConfigRegexpStr)
 )
 
-// Turn "service/method{h;m}" into "service", "method", "{h;m}".
-func parseMethodConfigAndSuffix(c string) (service, method, suffix string, _ error) {
+// ParseMethodConfigAndSuffix turns "service/method{h;m}" into "service",
+// "method", "{h;m}".
+func ParseMethodConfigAndSuffix(c string) (service, method, suffix string, _ error) {
```

Comment:
I don't think I would share this since the filters you're calling with won't have `{h:,m:}` stuff.  Can't you just split on "/" easily enough?